src/chat/ChatApp.js

```diff
@@ -0,0 +1,57 @@
+import Component from '../Component.js';
```

Comment:
somewhere in your component tree for your chat app it would be a nice user experience to be able to back to a list of chatrooms from your chatroom detail page.